mne/io/snirf/tests/test_snirf.py

```diff
@@ -186,6 +186,20 @@ def test_snirf_nirsport2():
     assert sum(short_channels(raw.info)) == 16
 
 
+@requires_testing_data
+@requires_h5py
+def test_snirf_coordframe():
+    """Test reading SNIRF files."""
+    assert read_raw_snirf(nirx_nirsport2_103, optode_frame="unknown").\
+        info['chs'][10]['coord_frame'] == 0
+    assert read_raw_snirf(nirx_nirsport2_103, optode_frame="head").\
+        info['chs'][10]['coord_frame'] == 4
+    assert read_raw_snirf(nirx_nirsport2_103, optode_frame="asdf").\
+        info['chs'][10]['coord_frame'] == 0
+    assert read_raw_snirf(nirx_nirsport2_103, optode_frame="mri").\
+        info['chs'][10]['coord_frame'] == 4
```

Comment:
why is it not

FIFF.FIFFV_COORD_MRI

? if there is a good reason can you put FIFFV_COORD_HEAD and explain why?

Reply:
> why is it not FIFF.FIFFV_COORD_MRI

Because we currently convert everything to head coordinates.

As of a year ago there wasn't even a way to store the coordinate frame of individual channels. Now there is (via the extended channel info blocks). In some/many places in MNE, we also still assume that non-MEG channels are in the HEAD coordinate frame. This is gradually being changed but we're not there yet.